### PR TITLE
Update UG after consultation

### LINKED ISSUE #195: [PE-D][Tester B] Insufficient Visuals in User Guide
Other than the screenshot provided under quick start, the user guide do not contain any other visuals to help users better understand how the other different features implemented works.

-------------
Labels: `severity.Low` `type.DocumentationBug`
original: dr-arrgghh/ped#3

### PR DESCRIPTION
**Pull request by Nephelite**
Fixes #195

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -300,10 +385,23 @@ Adds a task of type event into your task list.
...
+3. `task event o/Myself d/Upload Slides t/Jan 31 2022` adds an event called “Upload Slides” which is self-assigned by you.
+
+4. `task event d/Finish essay t/Jan 31 2022` adds an event called “Finish essay” which you assigned to yourself.
+
+<figure>
+<p>
+<p align = "center">
+<img src="images/ExampleEventAdd.png">
+</p>
+<figcaption align = "center"><b>Result of executng (1) to (4) sequentially. Indices may be different.</b>
```
Comment (dexter-sim) [inline docs/UserGuide.md:401]: I found a spelling here.

File: docs/UserGuide.md [hunk 2]
```diff
@@ -370,9 +488,29 @@ Format: `task edit i/INDEX <m/NAME o/NAME d/DESCRIPTION t/DATE #/TAG…>`
...
+<figcaption align = "center"><b>Initial state of 5 contacts.</b>
+</figcaption>
+</figure>
+
+<figure>
+<p>
+<p align = "center">
+<img src="images/ExampleTaskEditAfter.png">
+</p>
+<figcaption align = "center"><b>Result of executng (1) to (3).</b>
```
Comment (dexter-sim) [inline docs/UserGuide.md:511]: Same spelling here.

**Review by dexter-sim — CHANGES_REQUESTED** [review]

File: docs/UserGuide.md [hunk 3]
```diff
@@ -44,35 +44,76 @@ While reading this user guide, these symbols will inform you if there is some im
...
+These are the GUI components of TaskBook. Components are all described from left to right, top to bottom.
+<figure>
+<p align = "center">
+<img src="images/GuiTopHalf.png">
+</p>
+<figcaption align = "center"><b>Top half of the Graphical User Interface(GUI) of TaskBook.</b>
+</figcaption>
+</figure>
+
+1. **File** - A dropdown menu containing the exit button, which is an alternative to the [`bye`](#exiting-the-program--bye) [command](#using-the-examples-in-this-guide).
```
Comment (xiaobill8) [inline docs/UserGuide.md:63]: Maybe can replace with **Navigation Bar** that contains File and Help buttons

File: docs/UserGuide.md [hunk 4]
```diff
@@ -44,35 +44,76 @@ While reading this user guide, these symbols will inform you if there is some im
...
 <figure>
+<p align = "center">
 <img src="images/GuiExample.png">
-<figcaption align = "center"><b>The typical appearance of TaskBook.</b>
+</p>
+<figcaption align = "center"><b>Graphical User Interface(GUI) of TaskBook.</b>
+</figcaption>
+</figure>
+
+These are the GUI components of TaskBook. Components are all described from left to right, top to bottom.
```
Comment (xiaobill8) [inline docs/UserGuide.md:54]: top to bottom, left to right

File: docs/UserGuide.md [hunk 5]
```diff
@@ -430,7 +568,7 @@ Format: `contact find q/QUERY`
 * QUERY is not case-sensitive.
 
 Example:
-* `contact find q/ea` will change your displayed contact list to show a contact list where all contacts will have "ea" in their name.
+1. `contact find q/ea` will change your displayed contact list to show a contact list where all contacts will have "ea" in their name.
```
Comment (xiaobill8) [inline docs/UserGuide.md:571]: "Change" is a little awkward here, can avoid it completely. Consider rephrasing to "`contact find q/ea` will display all contacts with name containing "ea""

File: docs/UserGuide.md [hunk 6]
```diff
@@ -450,10 +588,13 @@ Format: `task find <q/QUERY x/DONE a/ASSIGNMENT>`
 </div>
 
 Examples:
-* `task find q/ea` will change the displayed task list to show a task list where all tasks will have "ea" in either their connected person, or their description.
-* `task find x/X` will change the displayed task list to show a task list where all tasks are marked done, with [X].
-* `task find a/FROM` will change the displayed task list to show a task list where all tasks are assigned to you by someone else.
-* `task find q/ea x/X` is equivalent to `task find x/X q/ea`.
+1. `task find q/ea` will change the displayed task list to show a task list where all tasks will have "ea" in either their connected person, or their description.
```
Comment (xiaobill8) [inline docs/UserGuide.md:591]: Consider rephrasing to "`task find q/ea` will display all tasks with description or corresponding contact name containing "ea""

File: docs/UserGuide.md [hunk 7]
```diff
@@ -450,10 +588,13 @@ Format: `task find <q/QUERY x/DONE a/ASSIGNMENT>`
...
+
+2. `task find x/X` will change the displayed task list to show a task list where all tasks are marked done, with [X].
```
Comment (xiaobill8) [inline docs/UserGuide.md:593]: Consider rephrasing to "`task find x/X` will display all tasks that are marked done."

File: docs/UserGuide.md [hunk 8]
```diff
@@ -450,10 +588,13 @@ Format: `task find <q/QUERY x/DONE a/ASSIGNMENT>`
...
+
+3. `task find a/FROM` will change the displayed task list to show a task list where all tasks are assigned to you by someone else.
```
Comment (xiaobill8) [inline docs/UserGuide.md:595]: Consider rephrasing to "`task find a/FROM` will display all tasks that are assigned to you."

File: docs/UserGuide.md [hunk 9]
```diff
@@ -333,8 +431,27 @@ Format: `contact edit i/INDEX <n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS #/TAG…>
...
+<figcaption align = "center"><b>Initial state of 2 contacts.</b>
+</figcaption>
+</figure>
+
+<figure>
+<p>
+<p align = "center">
+<img src="images/ExampleContactEditAfter.png">
+</p>
+<figcaption align = "center"><b>Result of executng (1) and (2).</b>
```
Comment (dexter-sim) [inline docs/UserGuide.md:452]:
```suggestion
<figcaption align = "center"><b>Result of executing (1) and (2).</b>
```

**Review by dexter-sim — CHANGES_REQUESTED** [review]

File: docs/UserGuide.md [hunk 10]
```diff
@@ -477,22 +622,27 @@ Alphabetical sorting will prioritize valid letters by their ASCII values, meanin
 </div>
 
 Example:
-* `contact sort s/a` will change the displayed contact list to show a contact list where names are sorted by alphabetical order. 
-  * Given 3 contacts: alfonzo, Alpha, and Betsy, the ordering will be: Alpha, Betsy, alfonzo.
+
+1. `contact sort s/a` will change the displayed contact list to show a contact list where names are sorted by alphabetical order.
```
Comment (xiaobill8) [inline docs/UserGuide.md:626]: Consider rephrasing to "`contact sort s/a` will display all contacts by ascending alphabetical order of names"

File: docs/UserGuide.md [hunk 11]
```diff
@@ -239,22 +296,36 @@ Adds a task of type todo into your task list.
...
+3. `task todo o/Myself d/Upload slides` adds a todo called “Upload slides” which is self-assigned by you.
+
+4. `task todo d/Finish essay` adds a todo called "Finish essay" which you assign to yourself.
+
+<figure>
+<p>
+<p align = "center">
+<img src="images/ExampleTodoAdd.png">
+</p>
+<figcaption align = "center"><b>Result of executng (1) to (4) sequentially. Indices may be different.</b>
```
Comment (dexter-sim) [inline docs/UserGuide.md:312]:
```suggestion
<figcaption align = "center"><b>Result of executing (1) to (4) sequentially. Indices may be different.</b>
```

File: docs/UserGuide.md [hunk 12]
```diff
@@ -502,8 +652,9 @@ Alphabetical sorting will prioritize valid letters by their ASCII values, meanin
 </div>
 
 Example:
-* `task sort s/a` will change the displayed task list to show a task list sorted by description alphabetical order.
-  * Given 3 tasks with descriptions: Walk the dog, Do homework, do chores, the order will: Do homework, Walk the dog, do chores.
+
+1. `task sort s/a` will change the displayed task list to show a task list sorted by description alphabetical order.
```
Comment (xiaobill8) [inline docs/UserGuide.md:656]: Consider rephrasing to "`contact sort s/a` will display all contacts by ascending alphabetical order of description"

**Review by xiaobill8 — COMMENTED** [review]
Consider rephrasing any instance of "will change the displayed task list to show" to simply "display [whatever is changed]"

File: docs/UserGuide.md [hunk 13]
```diff
@@ -300,10 +385,23 @@ Adds a task of type event into your task list.
...
 Examples:
-* `task event m/Alex Yeoh d/Finish user guide t/2022-12-31` adds an event called “Finish user guide” assigned by Alex Yeoh to you.
-* `task event o/Bernice Yu d/Finish the assignment t/Jan 31 2022 #/cs2103` adds an event called “Finish the assignment” tagged as "cs2103" which is assigned to Bernice Yu.
-* `task event o/Myself d/Upload Slides t/Jan 31 2022` adds an event called “Upload Slides” which is self-assigned by you.
-* `task event d/Finish essay t/Jan 31 2022` adds an event called “Finish essay” which you assigned to yourself.
+1. `task event m/Alex Yeoh d/Finish user guide t/2022-12-31` adds an event called "Finish user guide" assigned by Alex Yeoh to you.
+
+2. `task event o/Bernice Yu d/Finish the assignment t/Jan 31 2022 #/cs2103` adds an event called "Finish the assignment" tagged as "cs2103" which is assigned to Bernice Yu.
+
+3. `task event o/Myself d/Upload Slides t/Jan 31 2022` adds an event called “Upload Slides" which is self-assigned by you.
```
Comment (dexter-sim) [inline docs/UserGuide.md:392]:
```suggestion
3. `task event o/Myself d/Upload Slides t/Jan 31 2022` adds an event called "Upload Slides" which is self-assigned by you.
```

File: docs/UserGuide.md [hunk 14]
```diff
@@ -44,35 +44,76 @@ While reading this user guide, these symbols will inform you if there is some im
...
+</figure>
+
+{:start="4"}
+4. **Contact List** - A scrollable box that contains the list of contacts you have added to TaskBook. Odd numbered boxes are darker in colour than even numbered ones.<br>These are the fields of a contact.
+   * **Index** - The number TaskBook assigns to contacts you add. Can be reshuffled by the [`contact sort`](#sorting-contacts--contact-sort) [command](#using-the-examples-in-this-guide).
+   * **Name** - The name of the contact, situated right of the index.
+   * **Tags** - A series of light blue boxes. If a contact has no tags, this region will not display any boxes.
+   * **Phone Number** - The contact's phone number. If no phone number was added, this field will be `[No phone number]` instead.
+   * **Address** - The contact's address. If no address was added, this field will display `[No address]` instead.
+   * **Email** - The contact's email. If no email was added, this field will display `[No email]` instead.
```
Comment (dexter-sim) [inline docs/UserGuide.md:94]:
```suggestion
   * **Phone Number** - The contact's phone number. If no phone number were added, this field will be `[No phone number]` instead.
   * **Address** - The contact's address. If no address were added, this field will display `[No address]` instead.
   * **Email** - The contact's email. If no email were added, this field will display `[No email]` instead.
```

File: docs/UserGuide.md [hunk 15]
```diff
@@ -211,20 +256,32 @@ Format: `contact add n/NAME [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [#/TAG]…`
...
-**Assigned by** Format:  `task todo m/NAME d/DESCRIPTION [#/TAG]…`
+Format:
+
+1. **Assigned by** Format:  `task todo m/NAME d/DESCRIPTION [#/TAG]…`
 
-**Assigned to** Format:  `task todo o/NAME d/DESCRIPTION [#/TAG]…`
+2. **Assigned to** Format:  `task todo o/NAME d/DESCRIPTION [#/TAG]…`
 
-**Self-assigned** Format:  `task todo d/DESCRIPTION`
+3. **Self-assigned** Format:  `task todo d/DESCRIPTION`
```
Comment (dexter-sim) [inline docs/UserGuide.md:282]:
```suggestion
3. **Self-assigned** Format:  `task todo d/DESCRIPTION [#/TAG]…`
```

File: docs/UserGuide.md [hunk 16]
```diff
@@ -239,22 +296,36 @@ Adds a task of type todo into your task list.
...
-**Assigned by** Format: `task deadline m/NAME d/DESCRIPTION t/DATE [#/TAG]…`
+Format:
+
+1. **Assigned by** Format: `task deadline m/NAME d/DESCRIPTION t/DATE [#/TAG]…`
 
-**Assigned to** Format: `task deadline o/NAME d/DESCRIPTION t/DATE [#/TAG]…`
+2. **Assigned to** Format: `task deadline o/NAME d/DESCRIPTION t/DATE [#/TAG]…`
 
-**Self-assigned** Format: `task deadline d/DESCRIPTION t/DATE`
+3. **Self-assigned** Format: `task deadline d/DESCRIPTION t/DATE`
```
Comment (dexter-sim) [inline docs/UserGuide.md:326]:
```suggestion
3. **Self-assigned** Format: `task deadline d/DESCRIPTION t/DATE [#/TAG]…`
```

File: docs/UserGuide.md [hunk 17]
```diff
@@ -270,22 +341,36 @@ Adds a task of type deadline into the task list.
...
+Format:
 
-**Assigned to** Format: `task event o/NAME d/DESCRIPTION t/DATE [#/TAG]…`
+1. **Assigned by** Format: `task event m/NAME d/DESCRIPTION t/DATE [#/TAG]…`
 
-**Self-assigned** Format: `task event d/DESCRIPTION t/DATE`
+2. **Assigned to** Format: `task event o/NAME d/DESCRIPTION t/DATE [#/TAG]…`
 
-* Adds an event assigned by (`m/`) or assigned to (`o/`) a contact with a description and an event date. Optional to include tags.
+3. **Self-assigned** Format: `task event d/DESCRIPTION t/DATE`
```
Comment (dexter-sim) [inline docs/UserGuide.md:371]:
```suggestion
3. **Self-assigned** Format: `task event d/DESCRIPTION t/DATE [#/TAG]…`
```

**Review by dexter-sim — CHANGES_REQUESTED** [review]

File: docs/UserGuide.md (hunk 14 above)
Comment (Nephelite) [reply docs/UserGuide.md:94]: Original is correct. While zero is indeed plural, the was is with reference to the phone number noun, so it should be 'was' and not 'were'.

Comment (dexter-sim) [reply docs/UserGuide.md:94]: Thanks for clarifying!

File: docs/UserGuide.md [hunk 18]
```diff
@@ -370,9 +488,29 @@ Format: `task edit i/INDEX <m/NAME o/NAME d/DESCRIPTION t/DATE #/TAG…>`
...
+2. `task edit i/2 o/James Lee`
+
+3. `task edit i/5 t/2023-12-31`
+
+<figure>
+<p>
+<p align = "center">
+<img src="images/ExampleTaskEditBefore.png">
+</p>
+<figcaption align = "center"><b>Initial state of 5 contacts.</b>
```
Comment (dexter-sim) [inline docs/UserGuide.md:502]:
```suggestion
<figcaption align = "center"><b>Initial state of 5 tasks.</b>
```

**Review by dexter-sim — APPROVED** [review]
lgtm

**Pull request merged**